MoreLinq.Test/SubsetTest.cs

```diff
@@ -116,6 +118,19 @@ public void TestAllSubsetsExpectedResults()
                 Assert.That(subset, Is.EqualTo(expectedSubsets[index++]));
         }
 
+        /// <summary>
+        /// See issue #645
+        /// </summary>
+        [Test]
+        public void Test0SubsetIsEmptyList()
+        {
+            var sequence = Enumerable.Range(1, 4);
+            var actual = sequence.Subsets(0);
+            var expected = new[] {new int[0]};
+
+            CollectionAssert.AreEquivalent(expected, actual);
```

Comment:
Seems simpler to just say: `actual.AssertSequenceEqual(new int[0]);`

Reply:
`new[] {new int[0]};` not `new int[0]`.
Maybe I should keep the `var expected = ...` line and comment it.

Reply:
Addressed with 29dbde66e27725e172ea7a1e8e63cdb2cba65ca0.